cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -37,6 +37,7 @@ set (opm-core_DEPS
 	# DUNE dependency
 	"dune-common"
 	"dune-istl"
+	"opm-common REQUIRED"
```

Comment:
On 08/09/15 22:02, Andreas Lauser wrote:
>
> In cmake/Modules/opm-core-prereqs.cmake 
> <https://github.com/OPM/opm-common/pull/35#discussion_r38972584>:
>
> > @@ -37,6 +37,7 @@ set (opm-core_DEPS
> >  	# DUNE dependency
> >  	"dune-common"
> >  	"dune-istl"
> > +	"opm-common REQUIRED"
>
>     we cannot do REQUIRED on the opm-common dependency right now as
>     opm-common does not produce a library and all dependencies are
>     assumed to contain one.
>
> that's strange: opm-material also does not produce a library and ewoms 
> requires via REQUIRED. Maybe a test needs to be defined instead?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OPM/opm-common/pull/35/files#r38972584>.
>
sorry for the low precision level. i meant header files, not library. 
it's the include path that holds the NOTFOUND value.